Sources/Subprocess/Error.swift

```diff
@@ -113,6 +116,8 @@ extension SubprocessError: CustomStringConvertible, CustomDebugStringConvertible
             return "Failed to create output from current buffer because the output limit (\(limit)) was reached."
         case .asyncIOFailed(let reason):
             return "An error occurred within the AsyncIO subsystem: \(reason). Underlying error: \(self.underlyingError!)"
+        case .outputBufferLimitExceeded(let limit):
+            return "Output exceeds the limit (\(limit))"
```

Comment:
```suggestion
            return "Output exceeds the limit of (\(limit))"
```

limit of `x` - bytes, I assume?